reposilite-backend/src/generated/kotlin/com/reposilite/ReposiliteProperties.kt

```diff
@@ -0,0 +1,3 @@
+package com.reposilite
+
+const val VERSION = "3.0.0-alpha.26-SNAPSHOT"
```

Comment:
If that's guaranteed to be generated during build, shouldn't the `generated` directory be excluded from git?

Reply:
Yes, this could be excluded from git, the only disadvantage is that the ide shows an error after you clone the repository if you haven't build the project yet.

Reply:
I see, ok, let's keep it for now, maybe I'll add a note to this directory to describe why it's attached